hastory-cli/src/Hastory/Cli/Internal.hs

```diff
@@ -51,3 +50,15 @@ runDb dbAction = do
   SQL.runSqlite (T.pack . toFilePath $ hDb) $ do
     SQL.runMigration migrateAll
     dbAction
+
+runDb' ::
```

Comment:
it's an anti-pattern that we need this extra function. What's going on here?

Reply:
`runDb'` is the `runDb`, but *without* the `MonadThrow` constraint. Since this PR is getting to be massive, I decided to remove all mentions of `MonadThrow` from the `Sync.hs` module, but *not* from the rest of our codebase. 

In Issue #34, I plan to remove `MonadThrow` completely, which will mean that clients of `runDb` will instead use `runDb'` (which will allow us to do a rename so that we no longer have a prime version at all).

Reply:
Ok!